src/Administration/Resources/app/administration/src/module/sw-order/page/sw-order-list/index.js

```diff
@@ -480,37 +482,15 @@ export default {
         },
 
         getVariantFromPaymentState(order) {
-            let technicalName = order.primaryOrderTransaction?.stateMachineState.technicalName;
-
-            if (!Shopware.Feature.isActive('v6.8.0.0')) {
-                technicalName = order.transactions.last().stateMachineState.technicalName;
-            }
-
-            // set the payment status to the first transaction that is not cancelled
-            for (let i = 0; i < order.transactions.length; i += 1) {
-                if (
-                    ![
-                        'cancelled',
-                        'failed',
-                    ].includes(order.transactions[i].stateMachineState.technicalName)
-                ) {
-                    technicalName = order.transactions[i].stateMachineState.technicalName;
-                    break;
-                }
-            }
+            const technicalName = order.primaryOrderTransaction?.stateMachineState.technicalName;
```

Comment:
We need to keep the old behavior behind the feature flag check (BC)
```suggestion
            const technicalName = order.primaryOrderTransaction?.stateMachineState.technicalName;
            
            if (!Shopware.Feature.isActive('v6.8.0.0')) {
                technicalName = order.transactions.last().stateMachineState.technicalName;
                
                // set the payment status to the first transaction that is not cancelled
                for (let i = 0; i < order.transactions.length; i += 1) {
                    if (
                        ![
                            'cancelled',
                            'failed',
                        ].includes(order.transactions[i].stateMachineState.technicalName)
                    ) {
                        technicalName = order.transactions[i].stateMachineState.technicalName;
                        break;
                    }
                }
            }
```